UnitTests/BraintreeThreeDSecureTests/BTPaymentFlowDriver+ThreeDSecure_Tests.swift

```diff
@@ -19,10 +19,15 @@ class BTPaymentFlowDriver_ThreeDSecure_Tests: XCTestCase {
     func testPerformThreeDSecureLookup_sendsAllParameters() {
         let expectation = self.expectation(description: "willCallCompletion")
 
-        threeDSecureRequest.amount = 9.97
         threeDSecureRequest.nonce = "fake-card-nonce"
-        threeDSecureRequest.accountType = .credit
+        threeDSecureRequest.amount = 9.97
         threeDSecureRequest.versionRequested = .version2
+        threeDSecureRequest.dfReferenceId = "df-reference-id"
```

Comment:
I re-ordered some of these lines and added a few that were missing so that we match what's in the source code.